plugins/modules/vmware_dvswitch.py

```diff
@@ -189,6 +189,52 @@
             - '   folder: /folder1/datacenter1/network/folder2'
         required: False
         type: str
+    health_check:
+        description:
+            - Dictionary which configures the Net Flow for the Distributed Switch.
+        suboptions:
+            collector_ip:
+                type: str
+                description: The IP Address (IPv4 or IPv6) of the NetFlow collector.
+                default: ''
+            collector_port:
+                type: int
+                description: The Port of the NetFlow collector.
+                default: 0
+            observation_domain_id:
+                type: int
+                description: Identifies the information related to the switch.
+                default: 0
+            active_flow_timeout:
+                type: int
+                description: The time, in seconds, to wait before sending information after the flow is initiated.
+                default: 60
+            idle_flow_timeout:
+                type: int
+                description: The time, in seconds, to wait before sending information after the flow is initiated.
+                default: 15
+            sampling_rate:
+                type: int
+                description:
+                    - The portion of data that the switch collects.
+                    - The sampling rate represents the number of packets that NetFlow drops after every collected packet.
+                    - If the rate is 0, NetFlow samples every packet, that is, collect one packet and drop none.
+                    - If the rate is 1, NetFlow samples a packet and drops the next one, and so on.
+                default: 409
+            internal_flows_only:
+                type: bool
+                description: If True, data on network activity between vms on the same host will be collected only.
+                default: False
+        type: dict
+        default: {
```

Comment:
@mariolenz I think this is the problem. Those brackets don't fit well in a YAML structure.

Reply:
@goneri But it looks pretty much the same as

https://github.com/ansible-collections/community.vmware/blob/c673d7517eb3a49bbb83e29661ab65fc84cc61c1/plugins/modules/vmware_dvs_portgroup.py#L248-L260

and this works. Hmm...

@Nina2244 Could you try to remove the last comma so it's only `internal_flows_only=False`? Maybe this helps.

Reply:
@mariolenz So what should I do?

Reply:
> So what should I do?

Try:

```
        default: {
            'collector_ip': '',
            'collector_port': 0,
            'observation_domain_id': 0,
            'active_flow_timeout': 60,
            'idle_flow_timeout': 15,
            'sampling_rate': 4096,
            'internal_flows_only': False
        }
```

in the documentation block. Then it looks like in `vmware_dvs_portgroup`. I hope this will work.